service/ciphering/secretKeyCreator.go

```diff
@@ -0,0 +1,20 @@
+package ciphering
+
+import (
+	"crypto/sha256"
+)
+
+const keySize = 32 // length of secret key which is minimal to complete AES Encoding
+
+func PassToSecretKey(pass string) []byte {
+
+	h := sha256.New()
+	h.Write([]byte(pass))
+
+	encryptedKeyWord := h.Sum(nil)
+	if h.Size() < keySize {
+		panic("ALARM")
```

Comment:
Provide meaningful description of why it panics and proposed options to fix it. For example, in the next format, "cannot convert a to b: hash should be c".